README.md

```diff
@@ -93,3 +93,45 @@ docker run -v/tmp/.conan:/home/conan/.conan lasote/conangcc63 bash -c "conan ins
 
 This command is sharing ``/tmp/.conan`` as a shared folder with the conan home, so the Boost package will be built there.
 You can change the directory or execute any other command that works for your needs.
+
+
+Build, Test and Deploy
+======================
+
+To create all default images, this project uses a bunch of script in conan_docker_tools. These scripts run as an application,
```

Comment:
Introduce: The images are already built and uploaded to "lasote" dockerhub account, If you want to build your own images you can do it by...